### DISCUSSION TITLE
기능제안, 프로젝트 이름표시와 이름 순으로 정렬

### DISCUSSION THREAD

**Discussion by wafe** [mantech-oss/jira-search-helper#13] [Ideas]
프로젝트가 많은데 코드만 표시되니까 찾기 익숙치가 않아서 제안드립니다. 고맙습니다!

![image](https://user-images.githubusercontent.com/1501557/190290869-750fe53c-7f69-40fc-890c-07ccdf3368c0.png)

**Comment by wafe** [comment]
https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-projects/#api-rest-api-2-project-search-get 

다른 이슈에서 알려주신 프로젝트 목록 문서를 보니, 정렬 순서를 lastIssueUpdatedTime  으로 해도 유용할 것 같다는 생각이 듭니다.

> lastIssueUpdatedTime: Sorts by the last issue update time.

**Reply by wafe** [reply]
제 로컬에 설치된 확장기능 소스를 임시로 수정해서 테스트해보니, lastIssueUpdatedTime  는 효과가 없는 것으로 보입니다.  (회사에서 사용하는 JIRA 클라우드 상품)

**Reply by taeuk-gang** [reply]
> 제 로컬에 설치된 확장기능 소스를 임시로 수정해서 테스트해보니, lastIssueUpdatedTime 는 효과가 없는 것으로 보입니다. (회사에서 사용하는 JIRA 클라우드 상품)

https://github.com/mantech-oss/jira-search-helper/blob/a37b4852d15777907c838a9ad7333c4986d01b0c/src/content/apis/project-list.ts#L98

해당 부분 수정한 것으로 보이는데, UI로 정렬을 변경할 수 있는 것이 고려해볼게요.

> 평일에는 회사일이 있어 주말에 시간이 되면(중요ㅎㅎ) 확인해보겠습니다!

**Reply by taeuk-gang** [reply]
> 제 로컬에 설치된 확장기능 소스를 임시로 수정해서 테스트해보니, lastIssueUpdatedTime 는 효과가 없는 것으로 보입니다. (회사에서 사용하는 JIRA 클라우드 상품)

잉..? 작업 다했는데... 말씀하신 것 처럼 `orderBy` 파라미터 작동을 안하네요. 대신 검색을 넣어서 개선하는걸로....

### JIRA API가 작동을 안함...
![image](https://user-images.githubusercontent.com/26294469/192108116-a51d2115-3bf0-432b-b263-530adf66f98d.png)

### 대신 검색 추가

![image](https://user-images.githubusercontent.com/26294469/192108206-724324c8-a333-4394-9d52-a93fdf106371.png)

**Comment by taeuk-gang** [comment]
v1.1.0 업데이트하여 기능추가 되었습니다 🎉

**Reply by wafe** [reply]
감동입니다!
